.coderabbit.yaml

```
@@ -4,9 +4,10 @@
 instructions: |
     When reviewing PRs, please provide suggestions for:
 
-    1. **Conventional Commit Titles** (if not following https://www.conventionalcommits.org/):
+    1. **Conventional Commit Titles** (if not following https://www.conventionalcommits.org/ or
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)